### ISSUE TITLE
HTTP error 403 when downloading from Animixplay

### ISSUE THREAD

**Issue by dcdelacuesta** [justfoolingaround/animdl#165]
"I have upgraded the project to its latest version, checked the open issues for any duplicates, am not spamming multiple same or similar issues and **have not edited the code myself**. In any of those cases, a maintainer may **CLOSE** or **DELETE** the issue without any notices." 
Yes, this is the latest version and I am too scared to edit the code myself. 

From maintainer: Your title should answer at least 2 or 3 of 5W1H questions for fastest responses.

**Describe the bug**
My last download was 1 week ago. I downloaded all of Slime Isekai's 2nd season. Now I'm trying to download all of Arifureta's 2nd season and I cannot download any. It just says Client Error 403 Forbidden. I have tried all other providers to which I have changed in config from animixplay to gogo and finally, to all the others. I also tried to uninstall/install it and restarting my linux computer but it didn't fix it. What should I do? 

**Did the error occur in between an active task (`download` / `stream` / `grab`)?**
In the download.

**Screenshots [Optional]**
![animdl-help](https://user-images.githubusercontent.com/78127954/179690382-4ed9781a-bcec-462c-b366-df6cb97089da.png)

**Additional information**
- animdl version 1.6.19*
- The screenshot is set to animixplay, all other providers has the same error. 
- sorry mate, this is my first ever github post and it may seem messy.

**Comment by justfoolingaround** [comment]
Please note what you were downloading as well as any other streams seem to work just fine.

**Comment by dcdelacuesta** [comment]
> Please note what you were downloading as well as any other streams seem to work just fine.

I found out it's actually fixed within 4-6 days later after posting this. I actually also did a little research while waiting but will this happen again if I download multiple 12-24 episode worth of anime within an hour and a half, almost everyday? I thought I was blocked by animixplay because animdl is basically scraping right? That got me scared for a bit but at least it's back now.
